docs/source/Installation.md

```diff
@@ -18,7 +18,7 @@ In some systems you could find pre-installed older versions. Please check this a
 </div>
```

Comment:
we could also add in the requirements section:

* that we added this helper script (in the Notes)

* that, together with python, there are other dependencies to install